app/models/event.rb

```diff
@@ -29,6 +29,45 @@ def self.featured
     upcoming_events.order('start_date').first
   end
 
+  def stats
+    results = {}
+    results[:by_project] = featured_project_stats
+    results[:by_attendee] = attendee_stats
+    results[:total] = {}
+
+    github_metrics = [:prs, :commits, :issues, :forks]
+
+    github_metrics.each do |metric|
+      results[:total][metric] = 0
+
+      results[:by_attendee].each do |project, stats|
```

Comment:
Unused block argument - `project`. If it's necessary, use `_` or `_project` as an argument name to indicate that it won't be used.